crates/sovereign-sdk/full-node/db/sov-db/src/schema/tables.rs

```diff
@@ -57,6 +57,7 @@ pub const LEDGER_TABLES: &[&str] = &[
     SoftBatchByHash::table_name(),
     L2RangeByL1Height::table_name(),
     L2Witness::table_name(),
+    L2StateRoot::table_name(),
```

Comment:
let's try not to introduce another table for this :D

Reply:
https://github.com/chainwayxyz/citrea/issues/880